safe_transaction_service/history/utils.py

```diff
@@ -43,9 +43,10 @@ def clean_receipt_log(receipt_log: Dict[str, Any]) -> Optional[Dict[str, Any]]:
     :param receipt_log:
     :return:
     """
+
     parsed_log = {
         "address": receipt_log["address"],
-        "data": receipt_log["data"],
+        "data": receipt_log["data"].hex(),
```

Comment:
data was before python bytes?

Reply:
Data was an `HexStr` too, now it's parsed